### PR TITLE
[RUCSS] Fix modified column not being updated in some cases

### LINKED ISSUE #6229: Modified date isnot changed when change status / retries of RUCSS
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version =>3.15.2
- Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
The modified date isnot changed while status/retries change during the generation of used CSS

**To Reproduce**
1. Enable RUCSS
2. Use the following snippet to see retries
`if ( is_user_logged_in() )
	return;
$slug = 'hello-world'; // Change this to match the slug of the page you want to delay its loading.
if ( false === stripos( $_SERVER['REQUEST_URI'], $slug ) )
	return;

add_action( 'wp_loaded', function(){
	sleep( 60 ); // The number of seconds WordPress loading will be stalled.
});`
3. Access the URL in snippet i.e domain/hello-world => entry is added to used CSS table (notice modified date)
5. Refresh admin and notice database => changing status/retries is not changing the modified date

**Expected behavior**
Modified timestamp shall change when status/retries change

**Screenshots**
Screenshot from 2023-10-17 11-11-37
Screenshot from 2023-10-17 11-13-13
Screenshot from 2023-10-17 11-13-39

**Additional context**
- This will prevent switching failed to pending

**Acceptance Criteria (for WP Media team use only)**
- Whenever the row is updated, modified date should match the timestamp of the change
- It should include actions like injecting Used CSS, incrementing retries, changing the status of the URL

### PR DESCRIPTION
**Pull request by wordpressfan**
## Description

We found a bug in the columns definitions for RUCSS database table that prevents BerlinDB from identifying the modified column to be updated with any update in the table.

Fixes #6229 

## Type of change

*Please delete options that are not relevant.*

- Bug fix (non-breaking change which fixes an issue).

## Is the solution different from the one proposed during the grooming?

As groomed

# Checklists

## Generic development checklist

- [x] My code follows the style guidelines of this project, with adapted comments and without new warnings.
- [ ] I have added unit and integration tests that prove my fix is effective or that my feature works.
- [x] The CI passes locally with my changes (including unit tests, integration tests, linter).
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] If applicable, I have made corresponding changes to the documentation. *Provide a link to the documentation.*

## Test summary

- [x] I triggered all changed lines of code at least once without new errors/warnings/notices.
- [ ] I validated all Acceptance Criteria of the related issues. (If applicable, provide proof).
- [ ] I validated all test plan the QA Review asked me to.

### CODE REVIEW

**Review by mostafa-hisham — APPROVED** [review]
looks good

File: inc/Engine/Optimization/RUCSS/Database/Schemas/UsedCSS.php [hunk 1]
```diff
@@ -140,7 +140,7 @@ class UsedCSS extends Schema {
 			'name'       => 'modified',
 			'type'       => 'timestamp',
 			'default'    => '0000-00-00 00:00:00',
-			'created'    => true,
```
Comment (CrochetFeve0251) [inline inc/Engine/Optimization/RUCSS/Database/Schemas/UsedCSS.php:143]: @wordpressfan why are we removing created, don't we need both?

Comment (mostafa-hisham) [reply inc/Engine/Optimization/RUCSS/Database/Schemas/UsedCSS.php:143]: https://github.com/wp-media/wp-rocket/issues/6229#issuecomment-1787063540

**Review by CrochetFeve0251 — APPROVED** [review]

**Review by vmanthos — APPROVED** [review]
Working as expected! :+1:

**Pull request merged**